contracts/Utils/Validators.sol

```diff
@@ -0,0 +1,26 @@
+// SPDX-License-Identifier: BSD-3-Clause
+pragma solidity 0.8.13;
+
+/// @notice Thrown if the supplied address is a zero address where it is not allowed
+error ZeroAddressNotAllowed();
+
+/// @notice Thrown if the supplied value is 0 where it is not allowed
+error ZeroValueNotAllowed();
+
+/// @notice Checks if the provided address is nonzero, reverts otherwise
+/// @param address_ Address to check
+/// @custom:error ZeroAddressNotAllowed is thrown if the provided address is a zero address
+function ensureNonzeroAddress(address address_) pure {
+    if (address_ == address(0)) {
+        revert ZeroAddressNotAllowed();
+    }
+}
+
+/// @notice Checks if the provided value is nonzero, reverts otherwise
+/// @param value_ Value to check
+/// @custom:error ZeroValueNotAllowed is thrown if the provided value is 0
+function ensureNonzeroValue(uint256 value_) pure {
```

Comment:
We could move this validation to https://github.com/VenusProtocol/solidity-utilities/blob/develop/contracts/validators.sol, and remove this file from this repo

Reply:
Created this new PR in **solidity-utilities:**
https://github.com/VenusProtocol/solidity-utilities/pull/13
Once it is aprroved and new version is release we can use that.

Reply:
Done https://github.com/VenusProtocol/protocol-reserve/pull/9/commits/781890cde08cd698b1d500ca9ae13fca99bc6834